### PR TITLE
StringSifter #2829

### PR DESCRIPTION
**Pull request by dualfade**
please review. Thank you.

### CODE REVIEW

File: packages/stringsifter/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+pkgname=stringsifter
+pkgver=14.f4331a0
+pkgrel=1
+pkgdesc='Machine learning tool that automatically ranks strings based on their relevance for malware analysis.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='ihttps://github.com/fireeye/stringsifter'
+license=('Apache')
+depends=('python' 'python-virtualenv' 'python-numpy' 'python-scikit-learn' \
```
Comment (noraj) [inline packages/stringsifter/PKGBUILD:12]: python-virtualenv is not used please remove it

File: packages/stringsifter/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stringsifter
+pkgver=14.f4331a0
+pkgrel=1
+pkgdesc='Machine learning tool that automatically ranks strings based on their relevance for malware analysis.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='ihttps://github.com/fireeye/stringsifter'
```
Comment (noraj) [inline packages/stringsifter/PKGBUILD:10]: there is a `i` before the URL

File: packages/stringsifter/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+license=('Apache')
+depends=('python' 'python-virtualenv' 'python-numpy' 'python-scikit-learn' \
+  'python-joblib' 'python-pytest')
```
Comment (noraj) [inline packages/stringsifter/PKGBUILD:13]: `python-pytest` is only for the tests that we remove and don't package so remove this dependency

File: packages/stringsifter/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+build() {
+  cd $pkgname
+
+  python setup.py build
+}
+
+package() {
+  cd $pkgname
+
```
Comment (noraj) [inline packages/stringsifter/PKGBUILD:32]: ## Setup.py

There is a setup.py file so why not just using `python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build` ?

## jointscript

Also if you use the jointscript method the `build()` method is not used.
You forgot to remove the `tests` folder.

**Comment by noraj** [comment]
@noptrix you can leave me the reviews if you are busy :wink:

File: packages/stringsifter/PKGBUILD (hunk 2 above)
Comment (dualfade) [reply packages/stringsifter/PKGBUILD:10]: Good catch. Fixed.

File: packages/stringsifter/PKGBUILD (hunk 3 above)
Comment (dualfade) [reply packages/stringsifter/PKGBUILD:13]: removed.

File: packages/stringsifter/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/stringsifter/PKGBUILD:12]: removed

**Pull request merged**